### PR TITLE
Add resque-scheduler background job adapter

### PR DESCRIPTION
**Pull request by dsalahutdinov**

### CODE REVIEW

File: lib/isolator/adapters/background_jobs/resque-scheduler.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+Isolator.isolate :"resque-scheduler",
```
Comment (palkan) [inline lib/isolator/adapters/background_jobs/resque-scheduler.rb]: Let's use underscore name, i.e. `resque_scheduler`

**Comment by dsalahutdinov** [comment]
it's done

File: README.md [hunk 2]
```diff
@@ -110,6 +110,7 @@ Isolator has a bunch of built-in adapters:
 - `:active_job`
 - `:sidekiq`
 - `:resque`
+- `:resque-scheduler`
```
Comment (palkan) [inline README.md]: `resque_scheduler`

**Comment by dsalahutdinov** [comment]
✔️

**Pull request merged**